robottelo/ui/locators/model.py

```diff
@@ -252,7 +253,11 @@ def __repr__(self):
         if keys:
             msg.append("contains=%s" % str(keys))
```

Comment:
the keys are always in ASCII (this is as variables names) , so no problem we have problem with the value that can contain unicode